Dockerfile

```diff
@@ -31,7 +31,7 @@ ONBUILD RUN mix do deps.get --only prod
 # phoenix and phoenix_html JS dependencies are included from Hex packages
 ONBUILD COPY package.json /usr/src/app/
 ONBUILD RUN npm install
+ONBUILD COPY . /usr/src/app/
 ONBUILD RUN mix deps.compile --only prod
 
-ONBUILD COPY . /usr/src/app/
```

Comment:
I suggest to clean up it more. Like here - https://github.com/shanesveller/docker-phoenix-framework/issues/6